offer/src/main/java/org/shareio/backend/core/usecases/service/GetOffersByUserUseCaseService.java

```diff
@@ -40,9 +39,12 @@ public List<UUID> getOfferResponseDtoListByUser(UUID id) throws MultipleValidati
             throw new NoSuchElementException("No offers found for this user!");
         }
 
-        List<OfferSnapshot> offerSnapshotList = getOfferDtoList.stream().map(Offer::fromDto).map(Offer::toSnapshot).toList();
-        List<UUID> offerUUIDList = new ArrayList<>();
-        offerSnapshotList.forEach(offer -> offerUUIDList.add(offer.offerId().getId()));
-        return offerUUIDList;
+        return getOfferDtoList
+                .stream()
+                .map(Offer::fromDto)
+                .filter(offer -> !offer.getStatus().equals(Status.CANCELED))
```

Comment:
I sense kind of an issue here, on the SingleOffer Page only CREATED offers should be visible, while on MyAccount Page we want to have access to our RESERVED and FINISHED offers.